packages/sdk/src/actor/behaviors/buttonBehavior.ts

```diff
@@ -17,24 +17,20 @@ import { TargetBehavior } from './targetBehavior';
  * Button behavior class containing the target behavior actions.
  */
 export class ButtonBehavior extends TargetBehavior {
-	private _hover: DiscreteAction = new DiscreteAction();
-	private _click: DiscreteAction = new DiscreteAction();
-	private _button: DiscreteAction = new DiscreteAction();
+	private _hover: DiscreteAction<void> = new DiscreteAction();
```

Comment:
>DiscreteAction<void> [](start = 17, length = 20)

Given the number of `DiscreteAction<void>`  and `ActionHandler<void>`, it might be worth giving those template parameters a default value of `void`, e.g.: `export class DiscreteAction<ActionDataT = void>`

Reply:
Didn't even know you could do that in TS.  Learn something new every day.